website/content/docs/connect/ca/vault.mdx

```diff
@@ -181,124 +170,207 @@ The configuration options are listed below.
-certificates and Consul may periodically generate new intermediates for
+signed with the root certificate. The intermediate signs all leaf
+certificates, and Consul may periodically generate new intermediates for
 automatic rotation. Therefore, Consul requires write access to this path.
 
-If either path does not exist, then Consul will attempt to mount and
-initialize it. This requires additional privileges by the Vault token in use.
-If the paths already exist, Consul will use them as configured.
+For each path, if the path does not exist, Consul attempts to mount and initialize
+a PKI secrets engine at that path. For this operation to succeed,
+the provided [Vault token](#token) must have the [required Vault privileges](#vault-acl-policies).
+If the path does already exist, Consul uses the PKI secrets engine at that path as configured.
 
-## Vault ACL Policies
+### Configure Vault ACL policies ((#vault-acl-policies))
 
-Vault PKI can be managed by either Consul or by Vault. If you want to manually create and tune the PKI secret engines used to store the root and intermediate certificates, use Vault Managed PKI Paths. If you want to have the PKI automatically managed for you, use Consul Managed PKI Paths.
+The Vault CA provider requires a [Vault token](#token) with a specific set of Vault privileges
+depending on whether you would prefer to control mount configuration from Vault or to delegate
+that responsibility to Consul.
 
-### Vault Managed PKI Paths
+Use [Vault-managed PKI paths](#vault-managed-pki-paths) to obtain the following benefits:
+- Enables use of a root CA external to Consul by instantiating the PKI secrets engine at
+  [`RootPKIPath`](#rootpkipath) with an intermediate certificate
+- Retain full control over PKI mount creation, and over `RootPKIPath` mount configuration
 
-The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
-Consul is granted read-only access to the PKI mount points and the Root CA, but is
-granted full control of the Intermediate or Leaf CA for Connect clients.
+Otherwise, use [Consul-managed PKI paths](#consul-managed-pki-paths) to let Consul fully automate PKI management.
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+#### Define a policy for Vault-managed PKI paths ((#vault-managed-pki-paths))
 
-<CodeBlockConfig filename="vault-managed-pki-policy.hcl">
+To use Vault-managed PKI paths, you must first instantiate and configure PKI secrets engines at
+`RootPKIPath` and `IntermediatePKIPath`.
 
-```hcl
-# Existing PKI Mounts
-path "/sys/mounts" {
-  capabilities = [ "read" ]
-}
+Next, define the following Vault ACL policy that allows Consul to use those pre-existing PKI mounts in Vault.
+The policy grants Consul limited use of the root PKI engine
+and full control of the intermediate PKI engine to issue service mesh certificates.
```

Comment:
```suggestion
Next, define a Vault ACL policy that allows Consul to use the pre-existing PKI mounts in Vault.
The policy grants Consul limited use of the root PKI engine
and full control of the intermediate PKI engine to issue service mesh certificates.
```
Aren't we repeating the same information in the steps below? I think we can delete this paragraph and the one above it. The next paragraph is a good introduction to the procedure for building a suitable policy.

Reply:
Yeah. I think I wrote this before I had the summary paragraph above that compares the two options.